lexers/testdata/hare.actual

```diff
@@ -0,0 +1,66 @@
+use bufio;
```

Comment:
It's probably a good idea to have some test data that's a bit simpler and also uses more language constructs (e.g. enums, switch/match statements).